src/app/pages/storage/snapshots/snapshot-list/snapshot-list.component.ts

```diff
@@ -151,7 +151,9 @@ export class SnapshotListComponent implements OnInit {
 
   doRollback(item) {
     const warningMsg = T("<b>WARNING:</b> Rolling back to this snapshot will permanently delete later snapshots of this dataset. Do not roll back until all desired snapshots have been backed up!");
-    this.entityList.dialogService.confirm(T("Warning"), warningMsg, false, T('Rollback')).subscribe(res => {
+    const msg = T("<br><br>You're attempting to roll back to an early snapshot, would you like to rollback the snapshot <b>") + item.snapshot_name + '</b>( ' + item.creation + ' )?';
```

Comment:
As-is, it's hard to tell what this question is asking.  Based on the bug, I think it is trying to give more information on the snapshot being rolled back to.  So:
```suggestion
    const msg = T("<br><br>Roll back to snapshot <i>") + item.snapshot_name + '</i> from ' + item.creation + '?';
```
(Note: avoid contractions and the word "you" in prompts and other text.)